packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -469,6 +459,11 @@ class EditableText extends StatefulWidget {
   /// Defaults to false, resulting in a typical blinking cursor.
   static bool debugDeterministicCursor = false;
 
+  /// {@macro flutter.widgets.editable.selectionEnabled}
```

Comment:
Did I set up this macro correctly?

Reply:
Yes. The template appears in RenderEditable, so the macro tag should be `flutter.rendering.editable.selectionEnabled`.